ansible/roles/openshift_logging/templates/logging-fluentd.yaml.j2

```diff
@@ -29,7 +28,6 @@ data:
       @include configs.d/openshift/filter-kibana-transform.conf
       @include configs.d/openshift/filter-k8s-flatten-hash.conf
```

Comment:
This file is totally out of date, and should dropped for the openshift-ansible method of installing logging.